homeassistant/components/tuya/device_trigger.py

```diff
@@ -0,0 +1,86 @@
+"""Provides device triggers for Tuya."""
```

Comment:
We should not add device triggers for these, as this is provided for the `event` entities.

Let's remove them.

Reply:
Thanks for the quick review @frenck. I'm not sure I understand your comment. The built-in Home Assistant Event component doesn't configure any device triggers itself as it doesn't understand the semantics of the events unfortunately in the current form. This configures them just for the Tuya event entities, in this case the wireless switch. It allows the user to easily configure automations (which I guess is the sole point of such wireless switches). See

![device-trigger](https://github.com/user-attachments/assets/858d1bf8-b6c3-48d3-9f01-368ce0aad434)

Without this, users have to create automations by listening for `tuya_event` events, figure out the right values etc. which is very difficult for the average user.

Please let me know if I misunderstood your comment. I'm more than happy to iterate on this